### PR TITLE
Create user types page and rename the category

### PR DESCRIPTION
**Pull request by matthiasdebaat**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: gallery/src/pages/user-test/user-types.markdown [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: "User types"
+---
+
+We have defined three user types for Home Assistant. They are a lean segmentation of customers that helps us making decision throughout the product. User types differ from traditional personas in that the segmentation criteria aren’t demographic and don’t personify a group into a single character with a fictitious background story. 
+
+# Outgrowers
+
+Users that outgrow big tech smart home sollutions. It just needs to work with easy setup via an app.
```
Comment (zsarnett) [inline gallery/src/pages/user-test/user-types.markdown:9]:
```suggestion
Users that outgrow big tech smart home solutions. It just needs to work with easy setup via an app.
```

File: gallery/src/pages/user-test/user-types.markdown [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+# Tinkerers
+
+Technoid users in home networking and development that knows how to code.
+
+# Questioner
+
+Users who want a more advanced home automation, but need support to make it work.
```
Comment (zsarnett) [inline gallery/src/pages/user-test/user-types.markdown:17]:
```suggestion
Users who want more advanced home automation, but need support to make it work.
```

**Comment by balloob** [comment]
@matthiasdebaat your sidebar.js was invalid, you added the `.markdown` extension but don't have to.

**Review by balloob — APPROVED** [review]

**Comment by matthiasdebaat** [comment]
Wondering how that could happen? Only changed the content.

**Comment by balloob** [comment]
You did that in a526791e1d15dfb2cbed257dbb6ba6e32975bbc9 😉

**Pull request merged**